Library/UIPageViewController+Extensions.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+extension UIPageViewController {
+
+  public func isScrollEnabled(_ enabled: Bool) {
```

Comment:
Is this to disable the bounce on the page view controller? It's annoying that there isn't just a simple flag to do this, and doesn't feel that nice to iterate through subviews but I understand it's not such an easy one to solve.

But an alternative that I've used in the past is to return `nil` from the `UIPageViewControllerDataSource` methods `pageViewController(_:viewControllerBefore:)` and `pageViewController(_:viewControllerAfter:)`. So if that isn't too difficult to implement in this case it might be a nicer way to do it? Up to you.